example/src/lib.rs

```diff
@@ -37,11 +37,28 @@ enum Vehicle {
     Car { brand: String, color: String },
 }
 
+#[derive(Serialize, TS)]
+struct Point<T>
+where
+    T: TS,
+{
+    time: u64,
+    value: T,
+}
+
+#[derive(Serialize, TS)]
+struct Series {
+    points: Vec<Point<u64>>,
```

Comment:
This one deserves a test on its own. While `Point<u64>` now passes the tests, embedding it inside a vector actually causes subtle breakage (the generic argument is no longer passed to TS). This might be useful for a follow-up PR, as I felt getting all edge-cases right in one go is currently out of my league.